client/commands/proofs/get.go

```diff
@@ -111,15 +113,22 @@ type proof struct {
 	Data   interface{} `json:"data"`
 }
 
-// OutputProof prints the proof to stdout
-// reuse this for printing proofs and we should enhance this for text/json,
-// better presentation of height
-func OutputProof(info interface{}, height uint64) error {
-	wrap := proof{height, info}
-	res, err := data.ToJSON(wrap)
+// FoutputProof writes the output of wrapping height and info
```

Comment:
Very nice!

I like the generalization here that lets it work for http.ResponseWriter as well.